src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.RSA.cs

```diff
@@ -125,20 +124,19 @@ internal static partial class AppleCrypto
             out SafeSecKeyRefHandle pPublicKey,
             out SafeSecKeyRefHandle pPrivateKey)
         {
-            using (SafeTemporaryKeychainHandle tempKeychain = CreateTemporaryKeychain())
+            SafeSecKeyRefHandle keychainPublic;
+            SafeSecKeyRefHandle keychainPrivate;
```

Comment:
Since these are not keychains the variables could probably use a better name.

Reply:
Good catch. I will fix it up in a follow up PR.